monitor/metrics/checker.go

```diff
@@ -114,3 +128,25 @@ func (mc *Checker) Watch(ctx context.Context, peersF func(context.Context) ([]pe
 		}
 	}
 }
+
+// Failed returns true if a peer has potentially failed.
+// Peers that are not present in the metrics store will return
+// as failed.
+func (mc *Checker) Failed(pid peer.ID) bool {
```

Comment:
Because it could be used by components to check whether a peer is up before performing an operation. Not used that way currently but that is a valid use case for this function.

Reply:
ok

Reply:
I mean, if it's not used, it's like commented code, and you  always want to remove commented code :P but it's fine with me if it stays

Reply:
It is used...

Reply:
https://github.com/ipfs/ipfs-cluster/pull/714/files#diff-55cc63a73aaf01cf27b2e4d60f437805R47

Reply:
ahh sorry nvm I see what I have done, forgot that I added `FailedMetric`. I am still not sure that I want to remove it for the reason I stated above, but yes you are correct it is unused.

Reply:
it's ok